gob/k8s/linkerd.yml

```diff
@@ -9,19 +9,36 @@ data:
       port: 9990
     routers:
       - protocol: http
-        label: out
-        servers:
-          - port: 4180
-        interpreter:
-          kind: io.l5d.namerd
-          dst: /$/inet/namerd-sync.default.svc.cluster.local/4100
-        timeoutMs: 1000
-        responseClassifier:
-          kind: io.l5d.retryableRead5XX
-
-      - protocol: http
         label: in
         servers:
           - port: 4080
             ip: 0.0.0.0
         baseDtab: /http/1.1 => /$/inet/127.1/8080
+
+      - protocol: h2
+        experimental: true
+        label: incoming
+        servers:
+          - port: 4280
+            ip: 0.0.0.0
+        baseDtab: /h2 => /$/inet/127.1/8080
+
+      - protocol: h2
+        experimental: true
+        label: grpc
+        servers:
+        - port: 4180
+          ip: 0.0.0.0
+        dstPrefix: /grpc
+        timeoutMs: 1000
+        responseClassifier:
+          kind: io.l5d.retryableRead5XX
+        interpreter:
+          kind: io.l5d.namerd
+          dst: /$/inet/namerd-sync.default.svc.cluster.local/4100
+        identifier:
+          kind: io.l5d.headerPath
+        baseDtab: |
+          /srv/proto.GenSvc   => /srv/gen;
+          /srv/proto.WordSvc  => /srv/word;
+          /grpc => /srv;
```

Comment:
This router specifies both an `interpreter` and `baseDtab`. Maybe the dtab that you're adding in baseDtab should instead be defined in k8s/namerd/default.dtab?